netutils/ftpd/ftpd.c

```diff
@@ -1470,16 +1471,18 @@ static FAR char *ftpd_node2path(FAR struct ftpd_pathnode_s *node,
         {
           if (namelen <= 0)
             {
-              allocsize += sprintf(&path[allocsize], "/");
+              next += snprintf(&path[next], allocsize - next, "/");
```

Comment:
general notice: we should not add `snprintf` result as:
1. it can be -1
2. it can be bigger than size passed as second parameter.